inference-engine/tests_deprecated/behavior/shared_tests/plugin_tests/behavior_test_plugin_infer_request.hpp

```diff
@@ -42,10 +42,7 @@ TEST_P(BehaviorPluginTestInferRequest, SetEmptyConfig) {
 // Load incorrect network to Plugin to get executable network
 TEST_P(BehaviorPluginTestInferRequest, canNotLoadNetworkToGetExeNetworkWithoutWeights) {
     InferenceEngine::Core core;
-    CNNNetwork network = core.ReadNetwork(GetParam().model_xml_str, Blob::CPtr());
-
-    ASSERT_THROW(core.LoadNetwork(network, GetParam().device, GetParam().config),
-                 InferenceEngineException);
```

Comment:
why have you removed `LoadNetwork`?

Reply:
Because read network throws an error in this case.

Reply:
but it worked before. What was changed?

Reply:
The test tries to load network without weighs, but weights are required. 
At the current moment ReadNetwork throws an exeption and doesn't allow to create incorrect network without weights.